cmake/libmultiprocess.cmake

```diff
@@ -3,6 +3,16 @@
 # file COPYING or https://opensource.org/license/mit/.
 
 function(add_libmultiprocess subdir)
+  # Provide a more helpful message without pulling in CapnProtoConfig.cmake,
+  # which is included again by libmultiprocess. Skip the check when using the
+  # depends toolchain.
+  if(NOT (DEFINED CMAKE_TOOLCHAIN_FILE))
+    find_program(_CAPNP capnp)
+    if(NOT _CAPNP)
+      message(WARNING "Cap'n Proto (capnp) not found. Compile with -DENABLE_IPC=OFF if you do not need IPC functionality.")
```

Comment:
We might also want to mention that `doc/build-*.md` contains instructions for installing this new dependency, should they choose to not add `-DENABLE_IPC=OFF`